praw/models/reddit/subreddit.py

```diff
@@ -1545,6 +1545,50 @@ def __init__(self, subreddit):
         """Construct an instance of the Modmail object."""
         self.subreddit = subreddit
 
+    def conversations(self, after=None, limit=25, other_subreddits=None,
+                      sort=None, state=None):
+        """Return a list of conversations.
```

Comment:
While you said you didn't hit any limits, it's very likely that Reddit may introduce pagination in the future. As a result, can we instead yield conversations? That way we can transparently introduce pagination (if needed) without breaking PRAW's API.